src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml

```diff
@@ -0,0 +1,760 @@
+	<StaticResource x:Key="M3MaterialSecondaryFabForeground"
+	                ResourceKey="OnSecondaryContainerBrush" />
+	<StaticResource x:Key="M3MaterialSecondaryFabBackground"
+	                ResourceKey="SecondaryContainerBrush" />
+
+	<StaticResource x:Key="M3MaterialSecondaryFabDisabledBackground"
+	                ResourceKey="SystemControlTransparentBrush" />
+	<StaticResource x:Key="M3MaterialSecondaryFabDisabledForeground"
+	                ResourceKey="OnSurfaceDisabledBrush" />
+	<StaticResource x:Key="M3MaterialSecondaryFabPointerOverStateOverlay"
+	                ResourceKey="OnSecondaryContainerHoverBrush" />
+	<StaticResource x:Key="M3MaterialSecondaryFabFocusedStateOverlay"
+	                ResourceKey="OnSecondaryContainerFocusedBrush" />
+	<StaticResource x:Key="M3MaterialSecondaryFabPressedStateOverlay"
+	                ResourceKey="OnSecondaryContainerPressedBrush" />
+	<StaticResource x:Key="M3MaterialSecondaryFabDisabledStateOverlay"
+	                ResourceKey="OnSurfaceDisabledLowBrush" />
+	<!--#endregion-->
+
+	<!--#region Tertiary FAB Brushes-->
+	<StaticResource x:Key="M3MaterialTertiaryFabForeground"
+	                ResourceKey="OnTertiaryContainerBrush" />
+	<StaticResource x:Key="M3MaterialTertiaryFabBackground"
+	                ResourceKey="TertiaryContainerBrush" />
+
+	<StaticResource x:Key="M3MaterialTertiaryFabDisabledBackground"
+	                ResourceKey="SystemControlTransparentBrush" />
+	<StaticResource x:Key="M3MaterialTertiaryFabDisabledForeground"
+	                ResourceKey="OnSurfaceDisabledBrush" />
+	<StaticResource x:Key="M3MaterialTertiaryFabPointerOverStateOverlay"
+	                ResourceKey="OnTertiaryContainerHoverBrush" />
+	<StaticResource x:Key="M3MaterialTertiaryFabFocusedStateOverlay"
+	                ResourceKey="OnTertiaryContainerFocusedBrush" />
+	<StaticResource x:Key="M3MaterialTertiaryFabPressedStateOverlay"
+	                ResourceKey="OnTertiaryContainerFocusedBrush" />
+	<StaticResource x:Key="M3MaterialTertiaryFabDisabledStateOverlay"
+	                ResourceKey="OnSurfaceDisabledLowBrush" />
+	<!--#endregion-->
+
+	<!--#region Default FAB Resources-->
+	<Thickness x:Key="M3MaterialFabPadding">16</Thickness>
+	<x:Double x:Key="M3MaterialFabContentWidthOrHeight">18</x:Double>
+	<CornerRadius x:Key="M3MaterialFabCornerRadius">16</CornerRadius>
+	<!--#endregion-->
+
+	<!--#region Small FAB Resources-->
+	<Thickness x:Key="M3MaterialSmallFabPadding">8</Thickness>
+	<x:Double x:Key="M3MaterialSmallFabContentWidthOrHeight">18</x:Double>
+	<CornerRadius x:Key="M3MaterialSmallFabCornerRadius">12</CornerRadius>
+
+	<!--#endregion-->
+
+	<!--#region Large FAB Resources-->
+	<Thickness x:Key="M3MaterialLargeFabPadding">30</Thickness>
+	<x:Double x:Key="M3MaterialLargeFabContentWidthOrHeight">27</x:Double>
+	<CornerRadius x:Key="M3MaterialLargeFabCornerRadius">28</CornerRadius>
+	<!--#endregion-->
+
+	<x:Double x:Key="M3MaterialFabFontSize">14</x:Double>
```

Comment:
I see there are 2 group of FontSizes in themes/toolkit (global search: `FontSize">\d+</x:Double>`):
- 14 themse+toolkit
- 12 toolkit
- 8 themes
can we put them into a typography.xaml and named them properly instead of M3Material$(typename)FontSize?